sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -326,6 +327,17 @@ struct _pi_queue : _pi_object {
   ze_fence_handle_t ZeOpenCommandListFence = {nullptr};
   pi_uint32 ZeOpenCommandListSize = {0};
 
+  // specifies whether this queue is in-order or not.
+  const bool InorderQueue;
+
+  // keeps track of the event associated with the last enqueued command into
+  // this queue. this is used to add dependency with the last command and
+  // updated with the latest event each time a new command is enqueued.
+  pi_event *PreviousEvent;
+
+  // keeps track of the command list for which PreviousEvent was associated.
+  ze_command_list_handle_t ZeInOrderCommandList;
```

Comment:
You don't need this: use `_pi_event::ZeCommandList`

Reply:
yes, right. this is an extra code, never used. will remove it.